code/ship/shiphit.h

```diff
@@ -22,6 +22,8 @@ class object;
 
 #define MISS_SHIELDS		-1
 
+const float DEATHROLL_ROTVEL_CAP = 6.3f;
```

Comment:
Maybe add a comment explaining why this number was chosen.
I know that the old code didn't have one either, but since you changed it. you probably know the reason at the moment and that would help people like me, who don't know that part of the code in the future.

I think we have the compiler support for constexpr float DEATHROLL_ROTVEL_CAP = 6.3f;

Reply:
I'll move the existing comment and expand it. As for why 6.3, aka 1 revolution per sec? I think it's as simple as "it looked cool in retail FS2" :)  aka an artistic choice